source/docs/zero-to-robot/step-2/frc-game-tools.rst

```diff
@@ -4,193 +4,164 @@ Installing the FRC Game Tools
 =============================
 
 The FRC Game Tools contains the following software components:
-LabVIEW Update, FRC Driver Station, and FRC Utilities. The
-LabVIEW runtime components required for the Driver Station and
-Utilities are included in this package. **No components from the
-LabVIEW Software for FRC package are required for running either the Driver
-Station or Utilities.**
 
-.. note:: The Driver Station will only work on Windows 7, Windows 8,\
- Windows 8.1, and Windows 10. It will not work on Windows XP.
+- LabVIEW Update
+- FRC Driver Station
+- FRC Utilities.
+
+The LabVIEW runtime components required for the Driver Station and Utilities are included in this package.
+
+.. note:: No components from the LabVIEW Software for FRC package are required for running either the Driver Station or Utilities.
+
+Requirements
+------------
+
+- Windows 7 or higher (Windows 7, 8, 8.1, 10). Windows 10 is the recommended OS.
+- Download the update from `NI <https://www.ni.com/en-us/support/downloads/drivers/download.frc-game-tools.html>`__.
+
+.. image:: images/labview/offline-installer.png
+
+If you wish to install on other machines offline, do not click the :guilabel:`Download` button, click :guilabel:`Individual Offline Installers` and then click :guilabel:`Download`, to download the full installer.
```

Comment:
This sentence seems a bit clunky, the punctuation doesn't help. How about this phrasing:
```suggestion
If you wish to install on other machines offline, click :guilabel:`Individual Offline Installers` before clicking :guilabel:`Download` to download the full installer.
```